### PR TITLE
Bu 3 d geometries 6.0

### PR DESCRIPTION
**Pull request by JohannaOtt**
SVC-1737

### CODE REVIEW

File: annex-2-3/mappings/Buildings/3D/3A2INSPIRE_BU3D_v2.halex.alignment.xml [hunk 1]
```diff
@@ -418,7 +418,7 @@ _target{&#xD;
 		}&#xD;
 	}&#xD;
 	&#xD;
-	if(lod1MultiSurface || lod1Solid || lod1Terrain){&#xD;
+	if(lod1MultiSurface || lod1Solid) {&#xD;
```
Comment (annat2022) [inline annex-2-3/mappings/Buildings/3D/3A2INSPIRE_BU3D_v2.halex.alignment.xml:421]: groovy style: space is missing for `if(`

File: annex-2-3/mappings/Buildings/3D/3A2INSPIRE_BU3D_v2.halex.alignment.xml [hunk 2]
```diff
@@ -454,7 +454,7 @@ _target{&#xD;
 			}&#xD;
 		}&#xD;
 	}&#xD;
-	if(lod2MultiSurface || lod2Solid || lod2Terrain){&#xD;
+	if(lod2MultiSurface || lod2Solid) {&#xD;
```
Comment (annat2022) [inline annex-2-3/mappings/Buildings/3D/3A2INSPIRE_BU3D_v2.halex.alignment.xml:457]: groovy style: space is missing for `if(`

File: annex-2-3/mappings/Buildings/3D/3A2INSPIRE_BU3D_v2.halex.alignment.xml (hunk 1 above)
Comment (JohannaOtt) [reply annex-2-3/mappings/Buildings/3D/3A2INSPIRE_BU3D_v2.halex.alignment.xml]: Done

File: annex-2-3/mappings/Buildings/3D/3A2INSPIRE_BU3D_v2.halex.alignment.xml (hunk 2 above)
Comment (JohannaOtt) [reply annex-2-3/mappings/Buildings/3D/3A2INSPIRE_BU3D_v2.halex.alignment.xml]: Done

**Comment by JohannaOtt** [comment]
Thanks for the review

> If you do groovy style fixes explicitly in a separate commit already it would be great to do all fixes in this commit and not mixing them with the content related changes.

Fair point. Followed the original approach for the fixups now (changes concerning terrain intersection lines fully included in fix commit) but will keep it in mind for upcoming changes.

**Review by annat2022 — APPROVED** [review]
LGTM

**Pull request merged**